frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Box, Text, Link, Divider } from "@chakra-ui/react";
+
+const Footer = () : JSX.Element => {
+    const footerProps : {page: string, url: string}[] = [
+        {'page': 'Help', 'url':'/help'},
+        {'page': 'Propsective Mentors', 'url':'/mentors'},
+        {'page': 'Terms', 'url':'/terms'},
+        {'page': 'Privacy Policy', 'url':'/privacy'},
+    ];
+
+    const links : JSX.Element[] = [];
+    
+    footerProps.forEach((element,index) => {
```

Comment:
Super interesting solution here Tim! A more conventional way to achieve the same outcome is to map `footerProps` directly inside of the JSX:

```
return (
    {footerProps.map((item, index) => index === index.length -1 
          ?  <Text key={element.page+'div'}fontSize='10px'> | </Text>
          :  <Link key={element.page} fontSize='10px' href={element.url}>{element.page}</Link>
     )}
```

Reply:
I think the reason I chose to leverage forEach was because in the input array (footerProps) does not have pipes ( | ) to divide each link. And with the map method, I believe the input array and the output array need to have the same length. With forEach, I can create an array where each element in footerProps is separated by another element (pipe).

i.e. 

inputArray = [footerProps[0], footerProps[1], footerProps[2]]
outputArray = [<Link>, <Text> | </Text>, <Link>, <Text> | <T/ext>, <Link>, <Text> | </Text> ]